app/src/main/java/com/truongdc21/mediatree/utils/extension/ExtentionView.kt

```diff
@@ -0,0 +1,8 @@
+package com.truongdc21.mediatree.utils.extension
+
+import android.view.View
+import android.view.animation.AlphaAnimation
+
+fun View.setAnimationClick(){
+    this.startAnimation(AlphaAnimation(9f, 0.1f))
```

Comment:
ông nên define rõ tên method bằng loại animation , ví dụ fade out , slide in , slide left